resources/dicts/patrols/general/border.json

```diff
@@ -182,6 +182,29 @@
     "history_text": [
         "This cat got a scar from a fight with a small dog, but they always say it was a huge dog."
     ]
+},
+{
+    "patrol_id": "gen_bord_abandonedkittypet1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["border", "new_cat_injury_kittypet", "nc_sickness",  "respect", "warrior"],
```

Comment:
for the new_cat tags, you need to separate the injury tag and the attribute tag.  it should be "new_cat_injury", "new_cat_kittypet"